gfw_pixetl/layers.py

```diff
@@ -140,10 +112,12 @@ def input_files(self) -> List[Tuple[Polygon, str]]:
         input_files = list()
 
         o = urlparse(self._src_uri, allow_fragments=False)
```

Comment:
we should make this its own helper function `split_s3_uri(uri: str) -> Tuple[str,str]`